src/control/server/server.go

```diff
@@ -500,6 +500,10 @@ func waitFabricReady(ctx context.Context, log logging.Logger, cfg *config.Server
 
 // Start is the entry point for a daos_server instance.
 func Start(log logging.Logger, cfg *config.Server) error {
+	if err := common.CheckDupeProcess(); err != nil {
```

Comment:
should we also call this when calling any of the cmd/daos_server command "Execute()" methods, or even call from one of the base command embedded struct methods?

Reply:
Probably. Let's land this as-is and then evaluate what else could benefit from it. We know for certain that we don't want to support > 1 daos_server process on a node.